src/lib/y2partitioner/widgets/overview.rb

```diff
@@ -152,6 +152,22 @@ def validate
         valid_setup? && packages_installed?
       end
 
+      # Checks whether it is possible to jump to another page from the given page
+      #
+      # For example, when the current page is Hard Disks, we can go to the page of the selected disk.
+      #
+      # @param page [CWM::Page]
+      # @return [Boolean]
+      def allow_jump_from?(page)
```

Comment:
I find this a bit confusing. First of all the argument `page` is a bit unnecessary, the method is always called on `current_page` which is already known by `Overview`.

Even the name/semantic of the method is a bit strange. It basically returns false unless the current page corresponds to a concrete device. Isn't it?

Why not having a method called `device_page?` or something similar, without argument.

Reply:
You are absolutely right! It is much easier (and correct) to simply check whether the page has an associated device. Thanks!